core/trino-main/src/test/java/io/trino/sql/query/QueryAssertions.java

```diff
@@ -389,11 +389,19 @@ public QueryAssert returnsEmptyResult()
          */
         public QueryAssert isFullyPushedDown()
         {
-            checkState(!(runner instanceof LocalQueryRunner), "testIsFullyPushedDown() currently does not work with LocalQueryRunner");
```

Comment:
The new position of this is not equivalent. The `runner` may end up executing the query via `verifyResultsWithPushdownDisabled`.

I'm not sure if that is an issue though - will leave to others more knowledgeable about this.

Reply:
the check is because the code getting plans works with distributed only; result verification is hopefully fine